<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1316</width>
    <height>1112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ФВиЗ 1.0 beta</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="GraphView" name="graphicsView">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="tabletTracking">
       <bool>true</bool>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::AnchorUnderMouse</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="position">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>LT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Gk">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>Gk</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="name_izm">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>ед.изм</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1316</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_6"/>
    <addaction name="separator"/>
    <addaction name="action_9"/>
    <addaction name="action_10"/>
    <addaction name="separator"/>
    <addaction name="action_12"/>
    <addaction name="separator"/>
    <addaction name="action_7"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="action_3"/>
    <addaction name="separator"/>
    <addaction name="action_5"/>
    <addaction name="separator"/>
    <addaction name="action_14"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="action_8"/>
    <addaction name="separator"/>
    <addaction name="action_16"/>
    <addaction name="action_18"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="action_13"/>
    <addaction name="action_19"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_5"/>
   <addaction name="menu_4"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pics/pics/help.png</normaloff>:/pics/pics/help.png</iconset>
   </property>
   <property name="text">
    <string>Помощь</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Системные группы ФВ</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Список законов</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="action_12">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pics/pics/export.png</normaloff>:/pics/pics/export.png</iconset>
   </property>
   <property name="text">
    <string>Экспорт таблицы</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Параметры группы законов</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>Параметры закона</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>Настройки программы</string>
   </property>
  </action>
  <action name="action_17">
   <property name="text">
    <string>Оформление...</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Выйти</string>
   </property>
  </action>
  <action name="action_13">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pics/pics/undo.png</normaloff>:/pics/pics/undo.png</iconset>
   </property>
   <property name="text">
    <string>Отменить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_19">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pics/pics/redo.png</normaloff>:/pics/pics/redo.png</iconset>
   </property>
   <property name="text">
    <string>Повторить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="action_16">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pics/pics/+.png</normaloff>:/pics/pics/+.png</iconset>
   </property>
   <property name="text">
    <string>Приблизить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="action_18">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pics/pics/-.png</normaloff>:/pics/pics/-.png</iconset>
   </property>
   <property name="text">
    <string>Отдалить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphView</class>
   <extends>QGraphicsView</extends>
   <header>graphview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
